docs/openhabian.md

```diff
@@ -299,7 +299,7 @@ This function will rename the rules files so they get ignored by the starting op
 Starting with its general release, openHABian will install **openHAB 3** by default.
 There's some big changes also to openHABian such as to install Java 11 and to use changed file and directory names.
 Most directory names `... /openhab2/ ...` will become `... /openhab/ ...` (NOTE: not `openhab3`) plus there's changes in a number of places, often subtle ones like the name of Samba export shares to change.
-As openHABian users will be running openHAB 2.X by the time 3.0 gets released, `openhabian-config` offers to migrate the openHABian environment and install openHAB3 for you.
+For openHABian users running openHAB 2.X, `openhabian-config` offers to migrate the openHABian environment and install openHAB3 for you.
 Menu option 42 will do the upgrade. Be aware that it isn't the universal answer to all your migration needs - there is ONLY an openHAB upgrade path. You cannot downgrade from OH3 to OH2.
```

Comment:
(off the record) did you get this one ?

Reply:
For the most part, yes. Not the most elegant phrasing but it works.

Reply:
I was referring to the allusion to the Hitch Hiker's Guide ...